book/07-git-tools/sections/revision-selection.asc

```diff
@@ -207,6 +207,21 @@ Date:   Thu Dec 11 15:08:43 2008 -0800
     Merge commit 'phedders/rdocs'
 ----
 
+[NOTE]
+.CARET NEEDS TO BE QUOTED ON WINDOWS
```

Comment:
Let's not shout this too loudly. 😄 Change it to `.Escaping the caret on Windows`?

Reply:
Yes, that's better. I copied the convention from the existing note title ".A SHORT NOTE ABOUT SHA-1" in the document.